coil-core/src/androidInstrumentedTest/kotlin/coil3/decode/AndroidDecoderTest.kt

```diff
@@ -236,66 +252,84 @@ class BitmapFactoryDecoderTest {
 
     @Test
     fun exifOrientationPolicy_ignore() = runTest(timeout = 1.minutes) {
-        val factory = BitmapFactoryDecoder.Factory(
-            exifOrientationPolicy = ExifOrientationPolicy.IGNORE,
-        )
-
-        // Test JPG
-        for (index in 1..8) {
-            val assetName = "exif/$index.jpg"
-            val expected = BitmapFactory.decodeStream(context.assets.open(assetName))
-            val actual = decodeBitmap(assetName, Size.ORIGINAL, factory = factory)
-            assertTrue(expected.isSimilarTo(actual), "Image with index $index is incorrect.")
+        if (decoderFactory is BitmapFactoryDecoder.Factory) {
```

Comment:
Sorry for the misunderstanding - we can go back to using `assumeTrue(decoderFactory is BitmapFactoryDecoder.Factory` here

Reply:
Done